File: tests/test-artifacts/getresponsecode.js
var url = arguments[0];
var client = java.net.http.HttpClient.newHttpClient();
var request = java.net.http.HttpRequest.newBuilder().uri(java.net.URI.create(url)).GET().build();
var response = client.send(request, java.net.http.HttpResponse.BodyHandlers.ofString());
var statusCode = response.statusCode();
println(statusCode);

File: src/tests/test-artifacts/GetResponseCode.java
import java.net.http.HttpClient;
import java.net.http.HttpRequest;
import java.net.http.HttpResponse;
import java.net.URI;

public class GetResponseCode {
    public static void main(String[] args) throws Exception {
        String url = args[0];
        HttpClient client  = HttpClient.newHttpClient();
        HttpRequest request = HttpRequest.newBuilder().uri(URI.create(url)).GET().build();
        HttpResponse response = client.send(request, HttpResponse.BodyHandlers.ofString());
        System.out.println(response.statusCode());
    }
}